README.md

```diff
@@ -40,6 +48,24 @@ Generated sync functions are compatible with all Sync Gateway 1.x versions.
 
 **NOTE**: Due to a [known issue](https://github.com/couchbase/sync_gateway/issues/1866) in Sync Gateway versions up to and including 1.2.1, when specifying a bucket/database's sync function in a configuration file as a multi-line string, you will have to be sure to escape any literal backslash characters in the sync function body. For example, if your sync function contains a regular expression like `new RegExp('\\w+')`, you will have to escape the backslashes when inserting the sync function into the configuration file so that it becomes `new RegExp('\\\\w+')`. The issue has been resolved in Sync Gateway version 1.3.0 and later.
 
+### Validating
+
+To validate that your document definitions file is structured correctly and does not contain any obvious semantic violations, execute the built in validation script as follows:
+
+```
+node_modules/.bin/synctos-validate /path/to/my-document-definitions.js
```

Comment:
This is made possible by the inclusion of a [bin](https://docs.npmjs.com/files/package.json#bin) entry in `package.json`.